lambdas/call-ticf-cri/src/test/java/uk/gov/di/ipv/core/callticfcri/CallTicfCriHandlerTest.java

```diff
@@ -83,104 +86,38 @@ class CallTicfCriHandlerTest {
     @Mock private CiMitUtilityService mockCiMitUtilityService;
     @Mock private CriStoringService mockCriStoringService;
     @Spy private IpvSessionItem spyIpvSessionItem;
-    @Mock private SignedJWT mockSignedJwt;
+    @Mock private VerifiableCredential mockVerifiableCredential;
     @InjectMocks private CallTicfCriHandler callTicfCriHandler;
 
     @BeforeEach
-    public void setUp() {
+    public void setUp() throws ParseException, CredentialParseException {
         spyIpvSessionItem.setIpvSessionId("a-session-id");
     }
 
     @Test
     void handleRequestShouldCallTicfCriAndReturnJourneyNextIfNoBreachingCiReceived()
             throws Exception {
-        spyIpvSessionItem.setVcReceivedThisSession(VC_IN_STORE);
+        spyIpvSessionItem.setVcReceivedThisSession(VCS_RECEIVED_THIS_SESSION);
 
         when(mockIpvSessionService.getIpvSession("a-session-id")).thenReturn(spyIpvSessionItem);
         when(mockClientOAuthSessionDetailsService.getClientOAuthSession(any()))
                 .thenReturn(clientOAuthSessionItem);
-        when(mockUserIdentityService.getIdentityCredentials(any())).thenReturn(VC_IN_STORE);
-        when(mockTicfCriService.getTicfVc(clientOAuthSessionItem, spyIpvSessionItem, VC_IN_STORE))
-                .thenReturn(List.of(mockSignedJwt));
+        when(mockTicfCriService.getTicfVc(
+                        clientOAuthSessionItem, spyIpvSessionItem, VCS_RECEIVED_THIS_SESSION))
+                .thenReturn(List.of(mockVerifiableCredential));
 
         Map<String, Object> lambdaResult = callTicfCriHandler.handleRequest(input, mockContext);
 
         verify(mockCriStoringService)
                 .storeVcs(
                         TICF_CRI,
                         "an-ip-address",
-                        List.of(mockSignedJwt),
+                        List.of(mockVerifiableCredential),
                         clientOAuthSessionItem,
                         spyIpvSessionItem);
 
         verify(mockCiMitService)
-                .getContraIndicatorsVC(TEST_USER_ID, "a-govuk-journey-id", "an-ip-address");
-        verify(mockVerifiableCredentialService, times(1)).getVcStoreItems(TEST_USER_ID);
-
-        InOrder inOrder = inOrder(mockIpvSessionService);
-        inOrder.verify(mockIpvSessionService).updateIpvSession(spyIpvSessionItem);
-        inOrder.verifyNoMoreInteractions();
-
-        assertEquals("/journey/next", lambdaResult.get("journey"));
-    }
-
-    @Test
-    void handleRequestShouldOnlySendVcsReceivedInCurrentSession() throws Exception {
```

Comment:
Since removing logic between VcsReceivedThisSession and the "vcs to send", testing this behaviour is redundant as is identical to the test above